neo/NeoSystem.cs

```diff
@@ -34,10 +34,11 @@ public class NeoSystem : IDisposable
         public NeoSystem(Store store)
         {
             this.store = store;
+            Plugin.LoadPlugins(this);
             this.Blockchain = ActorSystem.ActorOf(Ledger.Blockchain.Props(this, store));
             this.LocalNode = ActorSystem.ActorOf(Network.P2P.LocalNode.Props(this));
             this.TaskManager = ActorSystem.ActorOf(Network.P2P.TaskManager.Props(this));
-            Plugin.LoadPlugins(this);
```

Comment:
You can keep `Plugin.LoadPlugins(this)` at here.

Reply:
The plugin must be loaded before `Blockchain` is created, or the Persist of the genesis block will not be caught by the plugin.

Reply:
The design of persistence plugins should include being able to be called for the Persistence of the genesis block.